connexion/__init__.py

```diff
@@ -26,5 +27,21 @@ def _required_lib(exec_info, *args, **kwargs):
 App = FlaskApp
 Api = FlaskApi
 
+if sys.version_info[0:2] >= (3, 4): # pragma: no cover
+    try:
+        from .apis.aiohttp_api import AioHttpApi
```

Comment:
The code above does almost exactly the same, except import paths. Let's refactor it to not repeat yourself.
```
try:
    from .apis.flask_api import FlaskApi
    from .apps.flask_app import FlaskApp
    from flask import request  # NOQA
except ImportError as e:  # pragma: no cover
    import six
    import functools

    def _required_lib(exec_info, *args, **kwargs):
        six.reraise(*exec_info)

    _flask_not_installed_error = functools.partial(_required_lib, sys.exc_info())

    FlaskApi = _flask_not_installed_error
    FlaskApp = _flask_not_installed_error

...

if sys.version_info[0:2] >= (3, 4): # pragma: no cover
    try:
        from .apis.aiohttp_api import AioHttpApi
        from .apps.aiohttp_app import AioHttpApp
    except ImportError as e:
        import six
        import functools

        def _required_lib(exec_info, *args, **kwargs):
            six.reraise(*exec_info)

        _aiohttp_not_installed_error = functools.partial(_required_lib, sys.exc_info())

        AioHttpApi = _aiohttp_not_installed_error
        AioHttpApp = _aiohttp_not_installed_error
```

I see it as
```
# We can make this as a high-order function and reuse
def not_installed_error():
    import six
    import functools

    def _required_lib(exec_info, *args, **kwargs):        
        six.reraise(*exec_info)

    return functools.partial(_required_lib, sys.exc_info())

...

try:
...
except ImportError:
    WhateverApi = not_installed_error()
```

This will make it more readable